app/backplane/power_module/CMakeLists.txt

```diff
@@ -4,11 +4,35 @@
 # Copyright (c) 2023 Launch Initiative
 # SPDX-License-Identifier: Apache-2.0
 
+set(BOARD_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+
+# Guarantee that prj.conf is the default and first configuration file to be read
+
+list(APPEND CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/core.conf)
+
+if (DEBUG)
```

Comment:
Is this just so we dont have to `-DOVERLAY_CONFIG=debug.conf`? or is there some other reason we want this in the CMakeLists as opposed to in the makefile?

Reply:
Yes to the first question. Just simpler and easier to remember doing a -DDEBUG instead of -DOVERLAY_CONFIG=blah which might not even exist if we copy pasta around